### PR TITLE
Fight UI

### PR DESCRIPTION
**Pull request by jponge**
Full-stack-ism in Action

### CODE REVIEW

**Comment by jponge** [comment]
Note: use the following properties to override backend endpoints:

```java
    @ConfigProperty(name = "endpoints.heroes", defaultValue = "http://localhost:8080/api/heroes/random")
    String heroesEndpoint;

    @ConfigProperty(name = "endpoints.villains", defaultValue = "http://localhost:8081/api/villains/random")
    String villainsEndpoint;

    @ConfigProperty(name = "endpoints.fight", defaultValue = "http://localhost:8082/api/fights")
    String fightEndpoint;
```

File: fight-ui/README.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Related Guides
+
+- Micrometer Registry Prometheus ([guide](https://quarkus.io/guides/micrometer)): Enable Prometheus support for Micrometer
+- Hibernate Validator ([guide](https://quarkus.io/guides/validation)): Validate object properties (field, getter) and method parameters for your beans (REST, CDI, Jakarta Persistence)
+- REST Jackson ([guide](https://quarkus.io/guides/rest#json-serialisation)): Jackson serialization support for Quarkus REST. This extension is not compatible with the quarkus-resteasy extension, or any of the extensions that depend on it
+- Hibernate ORM with Panache ([guide](https://quarkus.io/guides/hibernate-orm-panache)): Simplify your persistence code for Hibernate ORM via the active record or the repository pattern
+- SmallRye Health ([guide](https://quarkus.io/guides/smallrye-health)): Monitor service health
+- Kubernetes ([guide](https://quarkus.io/guides/kubernetes)): Generate Kubernetes resources from annotations
+- JDBC Driver - PostgreSQL ([guide](https://quarkus.io/guides/datasource)): Connect to the PostgreSQL database via JDBC
```
Comment (aureamunoz) [inline fight-ui/README.md:64]: Can we remove this? or just specify the fight-ui used extensions?

File: fight-ui/pom.xml [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+    <properties>
+        <compiler-plugin.version>3.13.0</compiler-plugin.version>
+        <maven.compiler.release>21</maven.compiler.release>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <quarkus.platform.artifact-id>quarkus-bom</quarkus.platform.artifact-id>
+        <quarkus.platform.group-id>io.quarkus.platform</quarkus.platform.group-id>
+        <quarkus.platform.version>3.12.1</quarkus.platform.version>
+        <skipITs>true</skipITs>
```
Comment (aureamunoz) [inline fight-ui/pom.xml:17]: Why skipping the IT? Do we have some ITs?

**Review by aureamunoz — COMMENTED** [review]
Also it would be great to add this module to dependabot.yml.
Apart from these minor changes, LGTM! great work!

File: fight-ui/README.md (hunk 1 above)
Comment (jponge) [reply fight-ui/README.md:64]: True!

File: fight-ui/pom.xml (hunk 2 above)
Comment (jponge) [reply fight-ui/pom.xml:17]: I'll be honest: I did a quick copy&paste of the POM, but yes this can be removed

**Pull request merged**